processor/elasticinframetricsprocessor/config.go

```diff
@@ -0,0 +1,9 @@
+package elasticinframetricsprocessor // import "github.com/elastic/opentelemetry-collector-components/processor/elasticinframetricsprocessor"
+
+type Config struct {
+	AddSystemMetrics bool `mapstructure:"add_system_metrics"`
```

Comment:
What does the `opentelemetry-lib` package do with the system metrics? Does it only add additional metrics? Or it also transforms actual ones?

I was wondering if maybe `translate_system_metrics` would be a more descriptive name.

Reply:
It adds system metrics, doesn't translate the existing ones.